packages/near-lake-framework/README.md

```diff
@@ -0,0 +1,37 @@
+# NEAR Lake Framework
+
+NEAR Lake Framework is a syntactical framework for accessing the NEAR Lake Indexer Data. It is a wrapper around the AWS API.
+
+Has `@near-lake/primitives` as a dependency, which contains the types for the data with associated helper functions.
+
+## Simple Example
+
+```ts
+import { startStream, types } from '@near-lake/framework';
+
+const lakeConfig: types.LakeConfig = {
+    s3BucketName: 'near-lake-data-mainnet',
+    s3RegionName: 'eu-central-1',
+    startBlockHeight: 66264389,
+};
+
+async function handleStreamerMessage(
+    block: types.Block,
+    context: types.LakeContext
+): Promise<void> {
+    let events = block.eventsByAccountId("x.paras.near")
```

Comment:
Adding a comment within the function will make it a bit more clear that this is the main place where the user of the library may add any code. 


```suggestion
    // custom logic for handling the block
    let events = block.eventsByAccountId("x.paras.near")
```